<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>网页开发助手</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>281</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::CustomContextMenu</enum>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFind"/>
   </widget>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionIndentIncrease"/>
   <addaction name="separator"/>
   <addaction name="actionHr"/>
   <addaction name="actionList"/>
   <addaction name="actionInsertLinks"/>
   <addaction name="actionBlod"/>
   <addaction name="separator"/>
   <addaction name="actionImage"/>
   <addaction name="actionRun"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_folder_open.png</normaloff>:/image/images/ic_folder_open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionIndentIncrease">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_format_indent_increase.png</normaloff>:/image/images/ic_format_indent_increase.png</iconset>
   </property>
   <property name="text">
    <string>indentIncrease</string>
   </property>
   <property name="toolTip">
    <string>Indent Increase</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_save.png</normaloff>:/image/images/ic_save.png</iconset>
   </property>
   <property name="text">
    <string>save</string>
   </property>
   <property name="toolTip">
    <string>Save File</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionInsertLinks">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_insert_link.png</normaloff>:/image/images/ic_insert_link.png</iconset>
   </property>
   <property name="text">
    <string>insertLinks</string>
   </property>
   <property name="toolTip">
    <string>Insert Links</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_wallpaper.png</normaloff>:/image/images/ic_wallpaper.png</iconset>
   </property>
   <property name="text">
    <string>image</string>
   </property>
   <property name="toolTip">
    <string>Convert to Base64</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_insert_drive_file.png</normaloff>:/image/images/ic_insert_drive_file.png</iconset>
   </property>
   <property name="text">
    <string>newFile</string>
   </property>
   <property name="toolTip">
    <string>New File</string>
   </property>
  </action>
  <action name="actionBlod">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_format_bold.png</normaloff>:/image/images/ic_format_bold.png</iconset>
   </property>
   <property name="text">
    <string>blod</string>
   </property>
   <property name="toolTip">
    <string>Blod</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_send.png</normaloff>:/image/images/ic_send.png</iconset>
   </property>
   <property name="text">
    <string>run</string>
   </property>
   <property name="toolTip">
    <string>Run</string>
   </property>
  </action>
  <action name="actionList">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_format_list_bulleted.png</normaloff>:/image/images/ic_format_list_bulleted.png</iconset>
   </property>
   <property name="text">
    <string>list</string>
   </property>
   <property name="toolTip">
    <string>List and Link</string>
   </property>
  </action>
  <action name="actionHr">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_remove.png</normaloff>:/image/images/ic_remove.png</iconset>
   </property>
   <property name="text">
    <string>hr</string>
   </property>
   <property name="toolTip">
    <string>HR</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_content_copy.png</normaloff>:/image/images/ic_content_copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionEditpast">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_content_paste.png</normaloff>:/image/images/ic_content_paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionEditcut">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_content_cut.png</normaloff>:/image/images/ic_content_cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionEditselectall">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_select_all.png</normaloff>:/image/images/ic_select_all.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
   <property name="toolTip">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionEditundo">
   <property name="text">
    <string>editundo</string>
   </property>
  </action>
  <action name="editredo">
   <property name="icon">
    <iconset resource="image.qrc">
     <normaloff>:/image/images/ic_redo.png</normaloff>:/image/images/ic_redo.png</iconset>
   </property>
   <property name="text">
    <string>editredo</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="image.qrc"/>
 </resources>
 <connections/>
</ui>
